sourcemash/frontend/templates/feed.html

```diff
@@ -15,7 +15,7 @@
 	            <div class="card-content black-text">
 	              <span class="card-title"><h4><a href="{{ item.link }}">{{ item.title }}</a></h4></span>
 	              	<p>{{ item.author }}</p>
-					<p>{{ item.summary }}</p>
+					<p>{{ item.text }}</p>
```

Comment:
We do want the summary, but I didn't want to make a new page for now since you'll be working on the backend changes.

So to make sure full text was being generated properly, I just put it on the feed page for now.